tests/simpleKVBC/ByzantineReplica/main.cpp

```diff
@@ -0,0 +1,94 @@
+// Concord
+//
+// Copyright (c) 2018-2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#include "setup.hpp"
+#include "ReplicaImp.h"
+#include "internalCommandsHandler.hpp"
+#include "replica_state_sync_imp.hpp"
+#include "block_metadata.hpp"
+#include "SimpleBCStateTransfer.hpp"
+#include "assertUtils.hpp"
+#include <csignal>
+
+#include "ByzantineReplicaImp.hpp"
+
+#ifdef USE_ROCKSDB
+#include "rocksdb/client.h"
+#include "rocksdb/key_comparator.h"
+#endif
+
+#include <memory>
+
+namespace concord::kvbc::test {
+
+std::atomic_bool timeToExit = false;
+
+void run_replica(int argc, char** argv) {
```

Comment:
As suggested above, maybe we could start simple by just having a main.cpp in order to instantiate `ByzantineKvbcReplicaImp` and reuse all the rest from the normal replica (incl. the internal commands handler).